src/README.md

```diff
@@ -12,54 +12,58 @@ author: poifuture
 > Make MS Word a markdown friendly collaborative editor.
 
 Welcome to markdown world!
-This MS Word add-in aims to apply MS Word styles to your document without changing your markdown content.
-You can easily view your document with a better style while collaborating with others on the document.
+This MS Word add-in aims to apply styles to your markdown document without changing your content.
+You can easily view your document with a better look while real-time collaborating with others.
 Our team is actively using it for writing our meeting notes.
 
 <!-- INSTALL SECTION BEGIN  -->
 
 ## Install
 
-Open MS Word Online -> Insert -> Office Add-ins -> Store -> Search "Markdown Styler" -> Add
+Open MS Word Online -> Insert" -> "Office Add-ins" -> "Store -> Search "Markdown Styler" -> Add
+
+If the add-in is successfully installed, the add-in will appear in the "Home" tab with a "Remark Selection" button.
 
 <!-- INSTALL SECTION END -->
 
 ## Usage
 
-1. Insert Readme and read the warning
-1. (Optional) Setup the document theme
-1. Click "Remark Document"
+1. Click "Option" in "Markdown Styler" section (click "..." if you don't find it)
```

Comment:
```suggestion
```
Remove because the only way to find this read me is to insert it into user's doc. We already have several pop ups to guide users to do so. (although sometimes the pop ups may get swallowed by Word)